website/addons/mendeley/static/node-cfg.js

```diff
@@ -0,0 +1,101 @@
+var $ = require('jquery');
+var ko = require('knockout');
+var $osf = require('osfHelpers');
+require('./node-cfg.css');
+
+var MendeleyAccount = function(display_name, id) {
+    var self=this;
+    self.display_name = display_name;
+    self.id = id;
+};
+
+var CitationList = function(name, provider_list_id, provider_account_id ) {
+    var self=this;
+    self.name = name;
+    self.provider_list_id = provider_list_id;
+    self.provider_account_id = provider_account_id;
+};
+
+var MendeleySettingsViewModel = function() {
+    var self=this;
+
+    self.settings_url = nodeApiUrl + 'mendeley/settings/';
```

Comment:
Prefer using `window.`contextVars.node.urls.api` over even more magical `nodeApiUrl`. As a personal preference, I also like using URI.js to build URLs (http://medialize.github.io/URI.js/). It's already in our JS dependencies.